java/src/jmri/jmrit/progsupport/ProgOpsModePane.java

```diff
@@ -56,6 +56,8 @@
     boolean oldOpsAccyMode = false;
     boolean opsSigMode = false;
     boolean oldOpsSigMode = false;
+    boolean lnAttachedBoardMode = false;
+    boolean oldLnAttachedBoardMode = false;
```

Comment:
PR looks good and I will approve it. But it might be good with a comment explaining what `lnAttachedBoardMode` is and what it's used for. Maybe by referring to the issue #10619?